config-model/src/main/java/io/strimzi/kafka/config/model/Scope.java

```diff
@@ -4,8 +4,22 @@
  */
 package io.strimzi.kafka.config.model;
 
+/**
+ * Scope of the configuration option
+ */
 public enum Scope {
+    /**
+     * Per-broker option
+     */
     PER_BROKER,
```

Comment:
when I did this on the bridge I preferred to have this kind of comments on one line to avoid the enum to be longer to scroll. But it's just my preference for readability.

Reply:
TBH, it seems better readable to me with the full comment.

Reply:
But you have to scroll more for the full list of enums in the editor. Anyway no need to change of course.